.codesandbox/ci.json

```diff
@@ -1,6 +1,11 @@
 {
```

Comment:
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30562&commitId=29e1de7ed4f70f67fe690c0af5b3a2a3a822eab9&env=prod&clientType=FLUENTUI)


  <details><summary><b>ComboBox</b> 3 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|ComboBox.Root - RTL.default.chromium.png|243| Changed|
|ComboBox.Root - RTL.hover.chromium.png|263| Changed|
|ComboBox.Root - RTL.click.chromium.png|1145| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v8"/>